testsuite/openexr-suite/run.py

```diff
@@ -50,3 +50,20 @@
 # Check writing overscan and negative range
 command += oiiotool("--create 64x64-16-16 3 -d half -o negoverscan.exr")
 command += info_command("negoverscan.exr", safematch=True)
+
+# Check ACES Container output for relaxed mode
+command += oiiotool("--create 4x4 3 -d half --compression none --ch B,G,R -sattrib oiio:ACESContainer relaxed -o relaxed-out.exr")
```

Comment:
I don't think you need to test B,G,R. No matter what order the channels are in a file, the exr reader will rearrange them to appear to be R,G,B,A first (if they exist), and no matter what order you start with, openexr itself will write it to the file in alphabetical order (this is an absurd thing for a file format to do, don't get me started). 

So I believe people understand that using --ch to reorder channels (without also renaming them) has no effect on exr files, and breaks just about everything else in OIIO (which assumes RGB without checking), so they don't do it. (But reordering and also renaming is a different matter, like "--ch R=B,G,B=R" actually does something useful.)